guardian/newsletters-nx
# Add statuses for async workflows and set them to a default on creation

## What does this change?

Adds four new properties to the newsletters data model

```
brazeCampaignCreationsStatus
ophanCampaignCreationsStatus
tagCreationsStatus:
signupPageCreationsStatus
```

These are set to `NOT_REQUESTED` at the point of creating a draft. 

The intent is that the values will be updated as those activities are completed and this will allow us to surface launch progress in the UI. 

In this first PR, the values are set to Optional. We can merge this PR, then backfill the existing draft & launched newsletters before creating a  final PR to set the property to required. 


## How to test

Create a newsletter (locally or when deployed) both using inMemory & s3 storage and then check that the values exist and are as expected.

## How can we measure success?

Values exist and are as expected

## Have we considered potential risks?

Nothing else in the app cares about these values so there is low risk in this change
